package.json

```diff
@@ -29,7 +29,7 @@
   "license": "MIT",
   "homepage": "https://github.com/companieshouse/overseas-entities-web#readme",
   "dependencies": {
-    "@companieshouse/api-sdk-node": "^2.0.110",
+    "@companieshouse/api-sdk-node": "github:companieshouse/api-sdk-node#feature/uar-315-trust-data-fetch-test3",
```

Comment:
temporary - for testing only. Will also revert packege-lock.json or update with merged api sdk changes.